app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/Invoices.php

```diff
@@ -14,6 +14,18 @@
 class Invoices extends \Magento\Framework\View\Element\Text\ListText implements
     \Magento\Backend\Block\Widget\Tab\TabInterface
 {
+    
+    /**
+     * Invoices constructor.
+     * @param \Magento\Framework\AuthorizationInterface|null $authorization
+     */
+    
+    public function __construct(
```

Comment:
Please consider that parent class has constructor
(\Magento\Framework\View\Element\AbstractBlock::__construct), please ensure you call it.